src/core/pointcloud/qgscopcpointcloudindex.cpp

```diff
@@ -408,4 +408,9 @@ QByteArray QgsCopcPointCloudIndex::fetchCopcStatisticsEvlrData()
   return statisticsEvlrData;
 }
 
+bool QgsCopcPointCloudIndex::gpsTimeFlag() const
+{
+  return mLazInfo.get()->header().global_encoding % 2;
```

Comment:
While this is correct, it is preferable to use bitwise AND operator to make it explicit that we want the first bit:
```suggestion
  return mLazInfo.get()->header().global_encoding & 1;
```